packages/ui/src/primitives/tailwind/Slider/index.tsx

```diff
@@ -35,12 +35,25 @@ export interface SliderProps {
   width?: number
   onChange: (value: number) => void
   onRelease: (value: number) => void
+  dataTestIdForValueInput?: string
+  dataTestIdForSlider?: string
 }
 
 /**
  * @param props.width width of the slider in pixels
  */
-const Slider = ({ value, min = 0, max = 100, step = 1, width = 200, onChange, onRelease, className }: SliderProps) => {
+const Slider = ({
+  value,
+  min = 0,
+  max = 100,
+  step = 1,
+  width = 200,
+  onChange,
+  onRelease,
+  className,
+  dataTestIdForValueInput,
```

Comment:
i'd very much prefer for us not to include these test IDs in our primitive library. can we keep them upstream in the UI?

Reply:
@HexaField So I went ahead and found another way to do this by "chaining" off of a more "parent" `data-testid` attribute like so:
```
  await page.getByTestId('files-panel-view-options-list-font-size-group').locator('input').first().fill('85');
  await page.getByTestId('files-panel-view-options-list-font-size-group').locator('input').nth(1).fill('54');
```

This test ID is now on the `<InputGroup>` component, which is defined in `packages/ui/src/components/editor/input/Group/index.tsx` and called in `packages/editor/src/panels/files/toolbar.tsx`.  

While it definitely works from our Playwright perspective, I still feel that in this case I'd want to be able to use the more robust `getByTestId()` method with a granular `data-testid` on the `<input>` itself since using the `nth()` locator relies on the _position_ of the elements to remain the same and so is much more subject to change as the UI gets worked on. We also lose context when working in the automation script as well as readability:

`await page.getByTestId('files-panel-view-options-list-font-size-group').locator('input').nth(1).fill('54')`
vs.
`await page.getByTestId('files-panel-view-options-list-font-size-slider-input').fill('54')`


Now, I know in this case I'm really splitting hairs with my argument, but I figured we're discussing how we move forward -- there are many more `data-testid` attributes that are going to get added so its best to nail the procedure down now.

Either way, let me know what you think 🙏🏽 .

Reply:
What if we did a middleground by adding generic IDs to the ui primitives, so we don't have to resort to magic numbers, but also don't have to pass in IDs explicitly.

Reply:
If I'm understanding correctly, is this what you mean?

```
      <input
        min={min}
        max={max}
        value={value}
        onChange={handleInputChange}
        onBlur={() => onRelease && onRelease(value)}
        className="h-8 w-14 rounded bg-neutral-900 text-center text-sm font-normal leading-[21px] text-neutral-400"
        data-testid="slider-text-value-input"
      />
```
      
     
So `data-testid` is now `slider-text-value-input` instead of `data-testid={dataTestIdForValueInput || ''`? 
      



      
This way, we don't include them as props here:
      
```
const Slider = ({
  value,
  min = 0,
  max = 100,
  step = 1,
  width = 200,
  onChange,
  onRelease,
  className,
  dataTestIdForValueInput, // here
  dataTestIdForSlider // here
}: SliderProps) => {

```

?

Reply:
Yup, so we can query for the higher level component, and then query into it for the desired inner element.